docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,19 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 21 December 2022
+Frontend version: 1.0.0-20221219.0756  | Backend version: 1.40.3-221219.0652
```

Comment:
sorry for last min change, the latest versions are: FE 1.0.0-20221221.0307 BE 1.40.4-221220.1019
```suggestion
Frontend version: 1.0.0-20221221.0307  | Backend version: 1.40.4-221220.1019
```